library/src/image.cxx

```diff
@@ -1,5 +1,6 @@
 #include "image.h"
 
+#include "export.h"
```

Comment:
why?

Reply:
we are using it below (F3D_NO_DEPRECATED). RN its coming from image.h but I think we should include what we need.

Reply:
It is needed in `image.h` tho because we are using `F3D_EXPORT` so it will never be needed here.

Reply:
untils its not and the ifndef logic breaks somewhere else in the file.

Reply:
Well, "until it's not" is not really an argument. We'll fix it when it's not.  
Anyway, not a big deal. Keep that change if you prefer.

Reply:
all I mean is that we should include what is being used in the file.

Reply:
I missed it thanks @mwestphal, it's fine for me like that